src/sentry/static/sentry/app/views/organizationIntegrations/providerRow.tsx

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line no-restricted-imports
```

Comment:
@NisanthanNanthakumar @scefali Can we make sure not to explicitly turn off this rule, we want to encourage people to remove usage of the library during pre-commit stage. It's fine to not refactor this in the PR (CI actually won't fail), but disabling the rule will defeat the purpose of the lint rule.

Reply:
@billyvg Makes sense, we can just skip the running the linter if we don't want to do the refactor.